commons-datastore/commons-datastore-core/src/main/java/org/opencb/commons/datastore/core/QueryOptions.java

```diff
@@ -37,6 +37,7 @@ public class QueryOptions extends ObjectMap {
     public static final String DESCENDING = "descending";
 
     public static final String TIMEOUT = "timeout";
+    public static final String NO_CURSOR_TIMEOUT = "noCursorTimeout";
```

Comment:
That's something too "MongoDB" related. Shouldn't it be next to `MongoDBCollection.BATCH_SIZE` ?

Reply:
I realised of those constants in that class after pushing. Just moved it there